connector/connect/server/src/main/scala/org/apache/spark/sql/connect/artifact/SparkConnectArtifactManager.scala

```diff
@@ -157,10 +159,46 @@ class SparkConnectArtifactManager private[connect] {
       }
     }
   }
+
+  private[connect] def uploadArtifactToFs(
+      sessionHolder: SessionHolder,
+      remoteRelativePath: Path,
+      serverLocalStagingPath: Path): Unit = {
+    val hadoopConf = sessionHolder.session.sparkContext.hadoopConfiguration
+    assert(
+      remoteRelativePath.startsWith(
+        SparkConnectArtifactManager.forwardToFSPrefix + File.separator))
+    val destFSPath = new FSPath(
+      Paths
+        .get("/")
+        .resolve(remoteRelativePath.subpath(1, remoteRelativePath.getNameCount))
+        .toString)
+    val localPath = serverLocalStagingPath
+    val fs = destFSPath.getFileSystem(hadoopConf)
+    if (fs.isInstanceOf[LocalFileSystem]) {
+      val allowDestLocalConf = sessionHolder.session.sparkContext.conf
+        .get("spark.fs.copyFromLocalToFs.allowDestLocal", "false")
```

Comment:
Used in testing.

Reply:
I think we should better add this conf to `org.apache.spark.sql.connect.config.Connect`, and explicitly document that it's for testing purpose.